nemoguardrails/rails/llm/config.py

```diff
@@ -1203,3 +1210,19 @@ def _generate_rails_flows(flows):
         flow_definitions.insert(1, _LIBRARY_IMPORT + _NEWLINE * 2)
 
     return flow_definitions
+
+
+def _is_file_ignored_by_railsignore(filename: str) -> bool:
+    # Default no skip
+    should_skip_file = False
```

Comment:
what do you think about renaming `should_skip_file` to `ignore`?

Reply:
ack, sounds good to me as well

Reply:
done